### PR TITLE
Right y axis fix #27590

### LINKED ISSUE #27590: First item of right Y-Axis legend is hard to click
When using right Y-axis, the first (rightmost) legend label is hard to click. Only a small fraction on the left side of the label is actually clickable. Almost impossible to hit on touch devices.

I assume this is due to the negative margin of the parent item (`<div class="view" ...>`)

recording

### PR DESCRIPTION
**Pull request by kkemppi**
**What this PR does / why we need it**: UX enhancement

**Which issue(s) this PR fixes**: #27590

**Special notes for your reviewer**:

Added suggested enhancement with some personal touch. Tested on graph visualization.
Feature was implemented by four member team in University course

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -55,19 +55,19 @@ RUN apk add --no-cache ca-certificates bash tzdata && \
...
     mkdir -p "$GF_PATHS_HOME/.aws" && \
     adduser -S -u $GF_UID -G "$GF_GID_NAME" grafana && \
     mkdir -p "$GF_PATHS_PROVISIONING/datasources" \
-             "$GF_PATHS_PROVISIONING/dashboards" \
-             "$GF_PATHS_PROVISIONING/notifiers" \
-             "$GF_PATHS_PROVISIONING/plugins" \
-             "$GF_PATHS_LOGS" \
-             "$GF_PATHS_PLUGINS" \
-             "$GF_PATHS_DATA" && \
+    "$GF_PATHS_PROVISIONING/dashboards" \
```
Comment (ryantxu) [inline Dockerfile:65]: This is unrelated... can you remove the file from the PR?

File: public/sass/components/_panel_graph.scss [hunk 2]
```diff
@@ -217,6 +217,12 @@
   }
 }
 
+.legend-yaxis-container {
```
Comment (ryantxu) [inline public/sass/components/_panel_graph.scss:220]: we are moving away from scss -- can you try doing this with emotion?

**Comment by dprokop** [comment]
@kkemppi thanks for your contribution. I'm gonna close this PR now as this issue should be now fixed in v8.
